app/classifier/index.cjsx

```diff
@@ -268,10 +268,29 @@ Classifier = React.createClass
         </p>}
     </div>
 
+  withinTolerance: (userX, userY, metaX, metaY, tolerance) ->
+    distance = Math.sqrt((userY - metaY)**2 + (userX - metaX)**2)
+    isWithinTolerance = distance < tolerance
+    return isWithinTolerance
+
```

Comment:
Looks like this has moved into the feedback component, so it isn't needed here any more?